1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,44 @@
 // Write your code here
+const items = document.querySelectorAll('img');
+const nextItem = document.querySelector('.next');
+const previousItem = document.querySelector('.previous');
+
+let count = 0;
+
+function showNextItem() {
+  items[count].classList.remove('active');
+  if(count < items.length - 1) {
+    count++;
+  } else {
+    count = 0;
+  }
```

Comment:
this block below
```js
  if(index < images.length - 1) {
    index++;
  } else {
    index = 0;
  }
```
can be simplified with the line beow:
```js
index = (index + 1) % images.length;
```

please try to debug it and see how it works.